### PR TITLE
copyedit c15

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Just checked over this. Looks good to go to me!

File: 15-synthesis.Rmd [hunk 1]
```diff
@@ -69,15 +69,15 @@ identical(nz_name1$Name, nz_name2$Name) # check results
...
 
-While commonly needed operators/functions (`[`/`select()` for example) were covered in depth, there are hundreds of other other functions for working with geographic data which have not been mentioned, let alone demonstrated, in the book.
+While commonly needed operators/functions were covered in depth --- such as the base R `[` subsetting operator and the **dplyr** function `filter()` --- there are many other functions for working with geographic data, from other packages, that have not been mentioned.
 Chapter \@ref(intro) mentions 20+ influential packages for working with geographic data, and only a handful of these are demonstrated in subsequent chapters.
 There are hundreds more.
-176 packages are mentioned in the Spatial [Task View](https://cran.r-project.org/web/views/) alone (as of October 2018);
-more packages and countless functions for geographic data are developed each year, making it impractical to do justice to all of them in a single book.
+As of early 2019, there are nearly 200 packages (183) mentioned in the Spatial [Task View](https://cran.r-project.org/web/views/);
```
Comment (jannes-m) [inline 15-synthesis.Rmd]: I suggest to use either 200 or 183 packages.

File: 15-synthesis.Rmd [hunk 2]
```diff
@@ -210,17 +210,18 @@ A next step in this case is to read-up on relevant articles in the area such as
 <!-- I think we want to be broader than that: being open source *enables* positive impact (RL) -->
 
 This is a technical book so it makes sense for the next steps, outlined in the previous section, to also be technical.
-However there are wider issues worth considering in this final section, which returns to our definition of geocomputation.
-One of elements of the term introduced in Chapter \@ref(intro) was that geographic methods should have a positive impact.
+However, there are wider issues worth considering in this final section, which returns to our definition of geocomputation.
+one of the elements of the term introduced in Chapter \@ref(intro) was that geographic methods should have a positive impact.
```
Comment (jannes-m) [inline 15-synthesis.Rmd]: Why have you put `one` to lowercase?

File: 15-synthesis.Rmd [hunk 3]
```diff
@@ -210,17 +210,18 @@ A next step in this case is to read-up on relevant articles in the area such as
...
+These considerations lead to the conclusion that geocomputation is part of a wider 'open source approach'.
+<!-- Engagement with this approach, and the community that supports it, can have tangible benefits. -->
 <!-- Consideration of real-world impacts leads to the conclusion that  -->
 
-As we saw in Section \@ref(what-is-geocomputation), other terms, including geographic information systems (GIS) and geographic data science (GDS), capture the range of possibilities opened-up by geospatial software.
-But geocomputation has advantages: it concisely captures the 'computational' way of working with geographic data advocated in this book --- implemented in code and therefore encouraging reproducibility --- and builds on desirable ingredients of its early definition [@openshaw_geocomputation_2000]:
+Section \@ref(what-is-geocomputation) presented other terms that mean roughly the same thing as geocomputation, including geographic information systems (GIS) and geographic data science (GDS).
```
Comment (jannes-m) [inline 15-synthesis.Rmd]: including GIScience and geographic data science.
GIS is just the tool but geocomputation, GIScience and GDS are more than just tools.

File: 15-synthesis.Rmd [hunk 4]
```diff
@@ -233,29 +234,30 @@ But geocomputation has advantages: it concisely captures the 'computational' way
...
-More than 600 issues have been closed, documenting a huge amount of work that has gone into making the package faster, more stable and user-friendly.
-Considering that **sf** is only one (relatively small) component in the wider R-spatial community provides a sense of the scale of the intellectual operation underway to make geocomputation with R possible at all, and continuously evolving.
+A quick glance at the `r-spatial/sf` GitHub repository, which hosts the code underlying the **sf** package, shows that 40+ people have contributed to the codebase and documentation.
+Dozens more people have contributed by asking question and by contributing to 'upstream' packages that **sf** uses.
+More than 600 issues have been closed on its [issue tracker](https://github.com/r-spatial/sf/issues), representing a huge amount of work to make **sf** faster, more stable and user-friendly.
+Considering that **sf** is only one component in the wider ecosystem of R's geographic offerings shows the scale of the intellectual operation underway to make geocomputation with R not only possible, but highly effective and continuously evolving.
```
[the remark this answers is not in the document]
Comment (jannes-m) [reply 15-synthesis.Rmd]: don't think that it is wrong but one stumbles across `R' s geographic offerings` or at least I do.

File: 15-synthesis.Rmd [hunk 5]
```diff
@@ -69,15 +69,15 @@ identical(nz_name1$Name, nz_name2$Name) # check results
 
 Again the question arises: which to use?
 Again the answer is: it depends.
-Each approach has advantages: the pipe syntax is popular and appealing to some, while base R is more stable, and is well-known to others.
+Each approach has advantages: the pipe syntax is popular and appealing to some, while base R is more stable, and is well known to others.
 Choosing between them is therefore largely a matter of preference.
 However, if you do choose to use **tidyverse** functions to handle geographic data, beware of a number of pitfalls (see the supplementary article [`tidyverse-pitfalls`](https://geocompr.github.io/geocompkg/articles/tidyverse-pitfalls.html) on the website that supports this book).
```
[the remark this answers is not in the document]
Comment (Robinlovelace) [reply 15-synthesis.Rmd]: No problem with that but not sure exactly the change you're proposing. Feel free to update the text though.

File: 15-synthesis.Rmd [hunk 6]
```diff
@@ -110,7 +110,7 @@ At the time of writing (October 2018) 69 packages depend on it, making it more t
 
 It is also being aware of promising alternatives that are under development.
 The package **stars**, for example, provides a new class system for working with spatiotemporal data.
-If you are interested in this topic, you can check for updates on the package's [source code](https://github.com/r-spatial/stars) and the broader [SpatialTemporal Task View](https://cran.r-project.org/web/views/SpatioTemporal.html).
+If you are interested in this topic, you can check for updates on the package's [source code](https://github.com/r-spatial/stars) and the broader [SpatiolTemporal Task View](https://cran.r-project.org/web/views/SpatioTemporal.html).
```
[the remark this answers is not in the document]
Comment (Robinlovelace) [reply 15-synthesis.Rmd]: Well spotted. Fixed.

File: 15-synthesis.Rmd (hunk 3 above)
Comment (Robinlovelace) [reply 15-synthesis.Rmd]: Good point. Changed.

**Pull request merged**